_posts/04-03-01-PEAR.md

```diff
@@ -57,7 +57,7 @@ handle your PEAR dependencies. This example will install code from `pear2.php.ne
 {% endhighlight %}
 
 The first section `"repositories"` will be used to let Composer know it should "initialize" (or "discover" in PEAR
-terminology) the pear repo. Then the require section will prefix the package name like this:
+terminology) the pear repo. Then the required section will prefix the package name like this:
```

Comment:
Good point. Thanks. Backticks have been put in place - https://github.com/codeguy/php-the-right-way/pull/681/commits/246d08a5389f8f195601d87f87891f307f906a45